hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -512,7 +512,10 @@ public function upgradeBundle() {
     $this->upgrade_1030();
     $this->upgrade_1032();
     $this->upgrade_1033();
+    $this->upgrade_1034();
+    $this->upgrade_1035();
```

Comment:
What are these methods? Why are you adding them here if the only method you added is upgrade_1037?

Reply:
There were implementations but no call to the upgrade function, causing some inconsistencies with the upgrade numbering. That was the reason i added those two.